test/langtools/jdk/javadoc/doclet/testIOException/TestIOException.java

```diff
@@ -50,6 +53,9 @@ public class TestIOException extends JavadocTester {
 
     public static void main(String... args) throws Exception {
         var tester = new TestIOException();
+        if(Platform.isRoot() && !tester.isWindows()) {
+            throw new SkippedException("root user has privileged will make this test fail.");
+        }
```

Comment:
```suggestion
        if (Platform.isRoot() && !tester.isWindows()) {
            throw new SkippedException("root user has privileges that will make this test fail");
        }
        var tester = new TestIOException();
```

Reply:
`isWindows()` is not static, so call this function need a instance `TestIOException` object, thus `var tester = new TestIOException();` should at before line 56.